flutter_appauth_platform_interface/pubspec.yaml

```diff
@@ -5,14 +5,13 @@ homepage: https://github.com/MaikuB/flutter_appauth/tree/master/flutter_appauth_
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.10.0"
+  flutter: ">=1.12.13+hotfix.5"
 
 dependencies:
   flutter:
     sdk: flutter
-  plugin_platform_interface: ^2.0.0
+  plugin_platform_interface: ^2.1.3
```

Comment:
missed this but could you also revert this? this change isn't necessary as using caret syntax allows pub to resolve to this 2.1.3 if it's the latest version anyhow

Reply:
Done